sagenb_export/nbextension/list_handler.html

```diff
@@ -18,10 +18,11 @@ <h2>Use the Jupyter notebook.</h2>
                 <tt>sage --notebook=jupyter</tt>
                 on the command line.
             </p>
-            <h2>The old notebook is now deprecated.</h2>            
+            <h2>The old notebook is now deprecated.</h2>
             <p>
-                The old Sage notebook only works with Python2 and is now deprecated.
-                It is still available as an optional package, that you may need to install using <tt>sage -i sagenb</tt>.
+                Starting from version 9.0, Sage is using Python3 by default.
+                The old Sage notebook only works for Sage using Python2 and is now deprecated.
+                The old Sage notebook is still available as an optional package, working only under Python2 and that you need to install using <tt>sage -i sagenb</tt>.
```

Comment:
I suggest this small edit:

`...Python2 and that you...` -> `...Python2. You...`